.github/workflows/examples-nxp.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
               run: |
                   scripts/run_in_build_env.sh "\
                       ./scripts/build/build_examples.py \
-                      --target nxp-k32w0-freertos-lighting-factory \
+                      --target nxp-k32w0-freertos-lighting-factory-log-progress \
```

Comment:
Is this supposed to be in this PR?

Reply:
Yes, k32w0 is low on flash and this makes CI fit by disabling detail log strings. Without this, this PR flash increases do not pass CI.

Reply:
Added a comment explaining.